contracts/modules/RecoveryManager.sol

```diff
@@ -145,9 +145,10 @@ contract RecoveryManager is BaseModule, RelayerModule {
      */
     function cancelRecovery(BaseWallet _wallet) external onlyExecute onlyWhenRecovery(_wallet) {
         RecoveryConfig storage config = recoveryConfigs[address(_wallet)];
-        emit RecoveryCanceled(address(_wallet), config.recovery);
-        guardianStorage.setLock(_wallet, 0);
         delete recoveryConfigs[address(_wallet)];
+        guardianStorage.setLock(_wallet, 0);
+
+        emit RecoveryCanceled(address(_wallet), config.recovery);
```

Comment:
Since we deleted the storage slot at position `recoveryConfigs[address(_wallet)]`, shouldn't `config.recovery` always return `0` ?

Reply:
Yes you're right.